Appsilon/shiny.gosling
# Fede.add example function

Closes #51 

## Changes description

Implemented a function to run examples in a more easier way
Updated README accordingly

## How to test
Checkout to this branch
1. `devtools::load_all`
2. `run_example("multiTrackApp")`

You can check any of the other examples, 

`  "circularLinearWithBrush", "circularVisualEncoding", "multiTrackApp", "sarsCov2", "staticCircularBar",
  "structuralVariant", "textAnnotation"`
